log/value_test.go

```diff
@@ -10,26 +10,26 @@ import (
 
 func TestValueBinding(t *testing.T) {
 	var output []interface{}
+
 	logger := log.Logger(log.LoggerFunc(func(keyvals ...interface{}) error {
 		output = keyvals
 		return nil
 	}))
-	logger = log.With(logger, "ts", log.Timestamp, "caller", log.Caller)
+	logger = log.With(logger, "ts", log.DefaultTimestamp, "caller", log.DefaultCaller)
 
-	time.Sleep(50 * time.Millisecond)
```

Comment:
I will figure out a smaller value that still works after I merge the changes from this discussion.